test/dhcp/dhcp_isc_subnet_service_initialization_test.rb

```diff
@@ -70,6 +71,7 @@ def test_managed_subnets_network_addresses
     @initialization.load_configuration_file(DHCPD_CONFIG)
     subnets =  @subnet_service.all_subnets
     assert_equal "192.168.122.0", subnets[0].network
+    assert_equal 9000, subnets[0].options[:interface_mtu]
```

Comment:
This does not match the test description, how about moving this to a separate test `test_managed_subnets_options`?

Reply:
Sure, done.